djlsp/parser.py

```diff
@@ -448,6 +453,22 @@ def get_tag_hover(self, line, character, match: Match):
                 )
         return None
 
+    def get_context_hover(self, line, character, match: Match):
```

Comment:
Could you add test for this hover, see other for reverence https://github.com/fourdigits/django-template-lsp/blob/main/tests/test_parser.py#L189-L210